### ISSUE TITLE
Flipper Devtools encountered an error during render

### ISSUE THREAD

**Issue by swrobel** [react-navigation/react-navigation#9850]
### Current behavior

[React Error link](https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=undefined&args[]=)

```
PLUGIN "REACT NAVIGATION" ENCOUNTERED AN ERROR DURING RENDER
Error: Minified React error #130; visit https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at Vg (file:///Applications/Flipper.app/Contents/Resources/app.asar/bundle.js:90:114890)
    at A (file:///Applications/Flipper.app/Contents/Resources/app.asar/bundle.js:90:54738)
    at x (file:///Applications/Flipper.app/Contents/Resources/app.asar/bundle.js:90:56018)
    at file:///Applications/Flipper.app/Contents/Resources/app.asar/bundle.js:90:58125
    at fi (file:///Applications/Flipper.app/Contents/Resources/app.asar/bundle.js:90:69374)
    at Bl (file:///Applications/Flipper.app/Contents/Resources/app.asar/bundle.js:90:109875)
    at bk (file:///Applications/Flipper.app/Contents/Resources/app.asar/bundle.js:90:101446)
    at ak (file:///Applications/Flipper.app/Contents/Resources/app.asar/bundle.js:90:101374)
    at Tj (file:///Applications/Flipper.app/Contents/Resources/app.asar/bundle.js:90:101227)
    at Lj (file:///Applications/Flipper.app/Contents/Resources/app.asar/bundle.js:90:98001)
    at file:///Applications/Flipper.app/Contents/Resources/app.asar/bundle.js:90:46741
    at _e.unstable_runWithPriority (file:///Applications/Flipper.app/Contents/Resources/app.asar/bundle.js:101:3887)
    at gg (file:///Applications/Flipper.app/Contents/Resources/app.asar/bundle.js:90:46516)
    at jg (file:///Applications/Flipper.app/Contents/Resources/app.asar/bundle.js:90:46686)
    at ig (file:///Applications/Flipper.app/Contents/Resources/app.asar/bundle.js:90:46619)
    at Hb (file:///Applications/Flipper.app/Contents/Resources/app.asar/bundle.js:90:119009)
```

<img width="1478" alt="Screen Shot 2021-08-20 at 12 06 18 PM" src="https://user-images.githubusercontent.com/157270/130281972-541d8e4d-b166-4542-8bb5-9c387567c9ff.png">

### Expected behavior

Plugin works

### Reproduction

plugin issue

### Platform

- [ ] Android
- [X] iOS
- [ ] Web
- [ ] Windows
- [ ] MacOS

### Packages

- [ ] @react-navigation/bottom-tabs
- [ ] @react-navigation/drawer
- [ ] @react-navigation/material-bottom-tabs
- [ ] @react-navigation/material-top-tabs
- [ ] @react-navigation/stack
- [ ] @react-navigation/native-stack

### Environment

| package                                | version |
| -------------------------------------- | ------- |
| @react-navigation/native               | 6.0.2
| @react-navigation/bottom-tabs          | 6.0.5
| @react-navigation/stack                | 6.0.7
| @react-navigation/devtools | 6.0.1
| react-native-safe-area-context         | 3.3.0
| react-native-screens                   | 3.5.0
| react-native-gesture-handler           | 1.10.3
| react-native                           | 0.64.2
| node                                   | 16.7.0
| yarn                                   | 1.2.11
| flipper | 0.103.0
| react-navigation flipper plugin | 1.3.4

**Comment by christophemenager** [comment]
I have the exact same error with same library versions, except for flipper, mine is 0.102.0

**Comment by exneval** [comment]
Also faced the same problem react-native-flipper 0.104.0

**Comment by sajaddp** [comment]
I also have this problem in version 0.108.0 (50.0.0)

**Comment by sajaddp** [comment]
I also have this problem 
```
Flipper: v0.108.0 (50.0.0)
React Navigation plugin for Flipper: v1.3.4
```

**Comment by darrylyoung** [comment]
I'm having the same problem while using the following React Navigation dependencies:

```
"react-native-flipper": "^0.108.0",
"@react-navigation/bottom-tabs": "^6.0.5",
"@react-navigation/devtools": "^6.0.1",
"@react-navigation/elements": "^1.1.0",
"@react-navigation/native": "^6.0.2",
"@react-navigation/native-stack": "^6.1.0",
"@react-navigation/stack": "^6.0.7",
```

If there's anything else I can share that may help, let me know.

**Comment by lranches-gemini** [comment]
Also seeing this issue as well on both iOS and Android.

**Comment by Titozzz** [comment]
After digging, issue is coming from using exports from flipper that are gone. In 0.92 there was VirtualList exported but it's not anymore for example and since Flipper version is set to "^0.92" in devDeps but "*" in peerDeps, it's broken with the latest versions. We'd need someone to make a major cleaning up.

**Comment by zhigang1992** [comment]
I've managed to get it fixed with DataList instead VirtualList

<img width="1805" alt="image" src="https://user-images.githubusercontent.com/1057756/134522124-be783f50-11bc-4637-a1d8-bfff7cf328d3.png">

**Comment by uloco** [comment]
Do you have an idea when this fix will be released? I needed to downgrade to flipper 0.92 to make it work again at the moment. Unfortunately the flipper desktop app does not use the specified `react-native-flipper` version in `package.json`

**Comment by swrobel** [comment]
@uloco this fix was released in the react-navigation flipper plugin 1.3.5. I don't actually know how to update flipper plugins, so I had to remove/reinstall, but it works now

**Comment by uloco** [comment]
@swrobel Oh ok thanks. I thought that the plugins auto update, but doesn't look like it does. Also I had a lot of issues getting flipper to accept the new version. needed to reinstall flipper and the plugin and restart my mac for this -.-

**Comment by chungweileong94** [comment]
> @uloco this fix was released in the react-navigation flipper plugin 1.3.5. I don't actually know how to update flipper plugins, so I had to remove/reinstall, but it works now

Apparently, Flipper won't auto-update any of the plugins. However, you can update it by going to the "Plugin Manager", under "Install Plugins". From there, you can see a update button for those plugins that has new update.
